Final Project/scraperTest.py

```diff
@@ -1,40 +1,72 @@
 """
 unit tests for scraper
 """
-
 #import response
 import unittest
-
+import requests
+import csv
+import os.path
+import re
+import time
+from bs4 import BeautifulSoup
+from selenium import webdriver
 import scraper as myScraper
 
+dataFilename='trainingCards.txt'
```

Comment:
@VinothkumarK24 Do you think it would be better if we set up a dummy file for testing instead of using the actual file we will be writing code with

Reply:
Test_trainingcards.txt created instead